src/OfficeToolPlus/Dictionaries/Languages/de-de.xaml

```diff
@@ -298,41 +298,41 @@
     <sys:String x:Key="Activate_KMS_Reset.Description">KMS-Hostnamen entfernen und KMS-Port auf Standard zurücksetzen.</sys:String>
     
     <!-- Licenses list -->
-    <sys:String x:Key="Activate_Licenses">Office licenses</sys:String>
-    <sys:String x:Key="Activate_License">Licenses list</sys:String>
-    <sys:String x:Key="Activate_License.Description">Manage installed licenses.</sys:String>
-    <sys:String x:Key="Activate_License_Refresh">_Refresh</sys:String>
-    <sys:String x:Key="Activate_License_Expand">_Expand all</sys:String>
-    <sys:String x:Key="Activate_License_Collapse">_Collapse all</sys:String>
-    <sys:String x:Key="Activate_License_Clear">C_lear list</sys:String>
-    <sys:String x:Key="Activate_License_Status.Unactivated">Unactivated</sys:String>
-    <sys:String x:Key="Activate_License_Status.Activated">Activated</sys:String>
-    <sys:String x:Key="Activate_License_Lic.Description">Description:</sys:String>
-    <sys:String x:Key="Activate_License_Lic.Key">Last 5 characters of key:</sys:String>
-    <sys:String x:Key="Activate_License_Lic.Period">Remaining period:</sys:String>
-    <sys:String x:Key="Activate_License_Lic.Error_Code">Error code:</sys:String>
-    <sys:String x:Key="Activate_License_Lic.Error_Description">Error description:</sys:String>
-    <sys:String x:Key="Activate_License_VL.Type">Activation type:</sys:String>
+    <sys:String x:Key="Activate_Licenses">Office-Lizenzen</sys:String>
+    <sys:String x:Key="Activate_License">Lizenzen Liste</sys:String>
+    <sys:String x:Key="Activate_License.Description">Installierte Lizenzen verwalten.</sys:String>
+    <sys:String x:Key="Activate_License_Refresh">_Aktualisieren</sys:String>
+    <sys:String x:Key="Activate_License_Expand">_Alle erweitern</sys:String>
+    <sys:String x:Key="Activate_License_Collapse">_Alle einklappen</sys:String>
```

Comment:
The access keys should be same as original. Such as `_Refresh` -> `Aktualisie_ren` and `_Expand all` -> `Alle _erweitern`. If not, you should set a new one according the rules: https://wpf-tutorial.com/control-concepts/access-keys/